python311/rmlist.txt

```diff
@@ -1 +0,0 @@
-/usr/local/lib/python3.11/dist-packages/tornado/test/test.key
```

Comment:
Why is this removed?

Reply:
Because, it's failing now. Hmm, perhaps it should be site-packages now, actually.

Let me try to change it. Why do you need to remove it anyway?

Reply:
yep that was it

Reply:
Some organizations do not accept containers which contain extraneous private key material; so its easy to slice it out.

Reply:
Then it's better to have it earlier.